Design files and Assembly Manual

Atenshin opened the pull request:
STEP and STL of CDA1T car files and the assembly manual.

Atenshin committed:
Current V1 Design Files

Work in progress

Atenshin committed:
Design updates

STEP and STL Files with updated steering

Atenshin committed:
Fixing Folders Pt. 1

Atenshin committed:
Fixed folder issue

Atenshin committed:
Updates to battery holder

Michael7371 reviewed (dismissed):
Looks good! Thanks for getting this PR together.

Atenshin committed:
Small updates and Guidance Doc

Updates to Rear shrouds.
Added Lidar clip.
Added Guidance for 3d Printing, Electrical, and Assembly

Atenshin committed:
Camera Mount

Added Camera Mount to STEP and STL.

Atenshin committed:
Added rear wheel spacing tool

Michael7371 committed:
adding operating environment docs

willjohnsonk commented on OE Design/Operating Environment.docx:
Let's convert this to a .pdf for consistency between docs. Also, space in name.

john-chrosniak replied:
If we aren't planning to add to this doc then I agree a pdf would be better, but if we plan to revise it may be better to use a doc or another editable file format.

willjohnsonk commented on Design Files/Drawings, Notes, and Guides/Car Assembly.pdf:
Nitpick, but since these are Github directories/docs could we replace every space in a file name with a more standard style guide like https://github.com/worldbank/template? Right now the spaces and other characters cause them to be interpreted weird when using a CLI

willjohnsonk replied:
I wouldn't worry about the .step files if they have a space, they mostly look formatted fine and will just be loaded into an app anyways.

willjohnsonk commented on Design Files/Drawings, Notes, and Guides/Guidance for 3d Printing, Electrical, and Assembly.pdf:
Same as comment above, let's simplify file names and limit special characters while replacing spaces

willjohnsonk commented on Design Files/Drawings, Notes, and Guides/Car Assembly.pdf:
Can we associate the related 3D file names with each page of the build guide? Maybe not labeled in the diagram but at least listed so people aren't searching for which of the smaller pieces to use.

willjohnsonk reviewed (changes requested):
Initial review mostly for doc formatting changes, will take another pass when I have more time

Atenshin committed:
Updates

Removed commas
Replaced spaces with '-' in document file names
Added Steering Washer
Added PPA-CF printing guidance to  Guidance for 3d Printing

Atenshin committed:
Merge branch 'design-files-v1' of https://github.com/usdot-fhwa-stol/cda1tenth-hardware into design-files-v1

Atenshin committed:
Operating Environment Doc

Changed Operating Environment document to pdf

Atenshin committed:
Directory Formatting

Replaced most relevant spaces with '-'.
Added parts list reference to Car Assembly document.

Atenshin committed:
Directory changes and RPLidar S3 support

Removed environment directory.
Sorted environment files into Environment-Print-Files and Drawings-Notes-and-Guides.
Added another Sensor Plate support for RPLidar S3.
Clarified what camera goes on the camera mount.

Atenshin committed:
Removed one layer of directory

Atenshin commented:
Let me know if the directory still isn't how you want it. This was my interpretation of the ideas you were proposing near the end of the standup.

willjohnsonk commented on Drawings-Notes-and-Guides/Car-Assembly.pdf:
The assembly .pdf is only showing as one page for me even when downloaded, confirm the other pages didn't get lost in the refactor

willjohnsonk commented on Drawings-Notes-and-Guides/Operating-Environment.pdf:
I think this should be removed and instead be on Confluence. This environment doesn't account for the exterior enclosure requirements needed to operate in the default 2D lidar config.

Atenshin committed:
Changes based on comments on 8/20

Operating Environment PDF deleted.
Car assembly PDF reexported.

afrye51 commented on Drawings-Notes-and-Guides/Guidance-for-3d-Printing.pdf:
It looks like there is a blank page on the end, and could you add the source files for these pdfs if possible?

Atenshin replied:
I had been exporting the PDFs a lot. For some reason, exporting to PDF does not display what I see in the Word document regarding spacing and page breaks, despite my checking each line's spacing. I've added the source files.

Atenshin committed:
Added source files for written documents

afrye51 commented:
Is there an assembly file that could be included as well? Not sure if there's a standard format

Atenshin commented:
I hadn't uploaded that because there is no standard format for that file. It's some Fusion specific file type (.f2d) that doesn't open in anything but Fusion. A quick Google search shows that I could export it as a DWG or DXF, but that still requires you guys to have AutoCAD, Solidworks, or Fusion to edit. A handful of programs like Adobe Illustrator can open the DWG file, but I'm unsure about the completeness of what it can load in and the ability to edit.

Atenshin committed:
ZEDM Camera mount
